### PR TITLE
Create Quarto site and setup GH Action

### PR DESCRIPTION
**Pull request by chrisdburr**
This PR creates a quarto site, in the `site` directory and adds a GH action to automatically republish the site on merges into `main`.

No code is currently executed in the Quarto site. Any Jupyter notebooks referenced exist only on the GH repo.

### CODE REVIEW

**Comment by aranas** [comment]
@chrisdburr This PR contains changes related to setting up the quarto page, correct?
Or are there any changes to the report text that I need to review?

File: site/index.qmd [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+Welcome to our Github repository's project pages.
+These pages were set up to help users navigate information pertainining to the TEA-DT Community Pulse Check.
+
+On this site you will find (among other things) the following:
+
+- Information about the TEA-DT project and the community pulse check
+- Details about our methodology
+- Results of the survey analysis
+- News and Updates
```
Comment (aranas) [inline site/index.qmd:16]: I suggest we explicitly add a "News and Updates" Section to signal where people can expect to see news in the future. Could be directly below here or as new site. We can mention upcoming Gemini Call and publication of blogpost/report once ready

```suggestion
- News and Updates

## News & Updates

- We'll give a brief roundup of the survey results during a DT Hub [Gemini Call](https://digitaltwinhub.co.uk/gemini-call/) on Dec 3rd 2024 - Join us!
```

**Comment by chrisdburr** [comment]
> @chrisdburr This PR contains changes related to setting up the quarto page, correct? Or are there any changes to the report text that I need to review?

No changes to your original `.md` files, but changes to the text as it is displayed in the Quarto site.

File: site/analysis.qmd [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+![Attitudes and Satisfaction Towards Assurance Practices. This figure presents the results from multiple questions assessing respondents' attitudes and satisfaction with assurance practices. The first three bars correspond to statements where respondents rated their level of agreement on a Likert scale from disagreeing (orange) to agreeing (green). The stacked bars show the percentage of respondents selecting each Likert rating, with colors coding for the type of response.The bottom bar reflects responses to the question, "How satisfied are you with how your team identifies and documents requirements, actions, and decisions in your assurance process?" Ratings ranged from very unsatisfied (purple) to very satisfied (blues). Percentages of "Neutral" responses are represented in grey (not displayed as bars).](images/report_plots/formatted_plots/attitudes.png){#fig-attitudes .lightbox}
+
+## Evaluating the Gemini Principles
+
+In addition to the current practices and attitudes, a further key motivation for this survey was to better understand how the use and adoption of the Gemini Principles.
```
Comment (aranas) [inline site/analysis.qmd:91]:
```suggestion
In addition to the current practices and attitudes, a further key motivation for this survey was to better understand the use and adoption of the Gemini Principles.
```

**Review by aranas — CHANGES_REQUESTED** [review]
I now had time to review your edits @chrisdburr , thank you very much for this work on the report. I think the context you have provided for some of the findings really helps understand the significance of the results! I have only corrected a small typo.

**Pull request merged**